01-js-intro-and-jquery/keypoints/1_11/resources/script/script.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Created by MaximilianoDaniel on 11/09/2015.
+ */
+
+var onPageReady = function () {
+    var processRequest;
+    var showError;
+    var showComplete;
+    var listLenght;
+    var listAlbums;
+    var showAlbum;
+
+    showAlbum = function(album){
+
+    };
+    showError = function(xmlHttpRequest,errorText,objectException) {
+        window.alert('error');
+    };
+
+    showComplete = function(xmlHttpRequest,result) {
+        window.alert('complete');
+    };
+
+    processRequest = function(result) {
+        listLenght = result.albums.items.length;
+        listAlbums = result.albums.items;
```

Comment:
why not call this albums instead of listAlbums? it's not needed to call the list list...